src/main/asciidoc/transactions.adoc

```diff
@@ -33,82 +41,257 @@ static class Config extends AbstractCouchbaseConfiguration {
+    }
 
-public void doSomething() {
- transactions.run(ctx -> {
-  ctx.insert(couchbaseClientFactory.getDefaultCollection(), "id", "content");
-  ctx.commit();
- });
 }
 ----
 ====
 
-== Object Conversions
+== Transactions with CouchbaseTransactionalOperator
 
-Since the transactions library itself has no knowledge of your spring data entity types, you need to convert it back and
-forth when reading/writing to interact properly. Fortunately, all you need to do is autowire the `MappingCouchbaseConverter` and
-utilize it:
+CouchbaseTransactionalOperator is used for reactive transactions. It is an important class for Couchbase Transactions in Spring Data as there is no Reactive TransactionManager for Couchbase.
+The class simplifies programmatic transaction demarcation and transaction exception handling.
+The central method is transactional(), supporting transactional wrapping of functional sequences code that. This operator handles the transaction lifecycle and possible exceptions such that neither the ReactiveTransactionCallback implementation nor the calling code needs to explicitly handle transactions.
+Typical usage: Allows for writing low-level data access objects that use resources such as database connections but are not transaction-aware themselves. Instead, they can implicitly participate in transactions handled by higher-level application services utilizing this class, making calls to the low-level services via an inner-class callback object.
 
-.Transaction Conversion on Write
+Note that a CouchbaseTransactionalOperator is available as a bean.
+If creating one explicitly, it must be created with the following (and not TransactionalOperator.create(manager)).
+.TransactionalOperator Construction
 ====
 [source,java]
 ----
-@Autowired
-MappingCouchbaseConverter mappingCouchbaseConverter;
+public static CouchbaseTransactionalOperator create(CouchbaseCallbackTransactionManager manager) 
+====
 
-public void doSomething() {
-  transactions.run(ctx -> {
+.Transaction Access Using TransactionalOperator.execute()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   Airline airline = new Airline("demo-airline", "at");
-   CouchbaseDocument target = new CouchbaseDocument();
-   mappingCouchbaseConverter.write(airline, target);
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
+    Flux<Person> result = txOperator.execute((ctx) -> rxCBTmpl.findById(Person.class).one(person.id())
+        .flatMap(p -> rxCBTmpl.replaceById(Person.class).one(p.withFirstName("Walt"))));
+    result.blockLast();
+----
+====
 
-   ctx.insert(couchbaseClientFactory.getDefaultCollection(), target.getId(), target.getContent());
+.Transaction Access Using transactional()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   ctx.commit();
-  });
-}
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
```

Comment:
same here, is this meant to use txOperator?

Reply:
reactiveCouchbaseTemplate